assets/js/swap-phrase.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+(function() {
+  function init(field) {
+    fetch("/api/phrase")
+      .then(response => response.json())
+      .then(json => field.innerHTML = json.data);
+  }
+
+  let button = document.querySelector(".js-generate-phrase");
```

Comment:
If you don't intend to reassign this variable, you can use `const` instead of let.